tests/exploreServicesBreakDown.spec.ts

```diff
@@ -1817,8 +1814,12 @@ test.describe('explore services breakdown page', () => {
       await page.getByLabel('Remove line filter').click();
       // Enable regex - should not trigger empty query
       await page.getByLabel('Enable regex').click();
+      await expect(page.getByLabel('Enable regex')).toHaveCount(0);
```

Comment:
noticed these were flaking sometimes due to the debounce on this input. Although ideally we would debounce the query execution instead of the input